projects/fsastorefrontlib/src/core/icon-config/icon-config.ts

```diff
@@ -10,11 +10,12 @@ export const iconConfig: IconConfig = {
     symbols: {
       PROPERTY: 'fs-icon icon-house',
       AUTO: 'fs-icon icon-auto',
-      PERSON: 'fs-icon icon-heart',
+      HEART: 'fs-icon icon-heart',
```

Comment:
Renamed `person` icon to `heart`, so person icon can be used in the right context. We were not using heart icon anywhere anyway.